datasets/de/abgeordnetenwatch/crawler.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import json
+import math
+from zavod import Context
+from zavod import helpers as h
+
+
+def crawl(context: Context):
+    # Fetch the source data URL specified in the metadata to a local path:
+    api_response = context.fetch_json(context.dataset.data.url, cache_days=30)
+    total_results = api_response["meta"]["result"]["total"]
+    num_batches = math.ceil(total_results / 100)
+
+    bi = 0
+    while bi < num_batches:
+        context.log.info(
+            "Get Politicians from range {} to {}".format(bi * 100, (bi + 1) * 100)
+        )
+        if bi > 0:
+            api_response = context.fetch_json(
+                context.dataset.data.url,
+                params={"range_start": bi * 100},
+                cache_days=30,
+            )
+            batch_total_results = api_response["meta"]["result"]["total"]
+            if batch_total_results != total_results:
+                num_batches = math.ceil(batch_total_results / 100)
+
+        for candidate in api_response.get("data"):
+            parliament_info = candidate["parliament_period"]["label"].split(" ")
+            state = parliament_info[0]
+            inception_date = parliament_info[1]
+            dissolution_date = parliament_info[3]
+
+            position = h.make_position(
+                context,
+                name="Member of Parliament",
+                country="Germany",
+                subnational_area=state,
+                inception_date=inception_date,
+                dissolution_date=dissolution_date,
```

Comment:
It's best to remove these parameters. These parameters and properties are intended to represent the start of the existince of the position, more than the parliamentary term. So if you want to encode that you could be a member of landtag X from 1975, you could use this.  I'll try and clarify the docs to this effect.

We don't distinguish parliamentary terms right now, but we do capture when a person enters and leaves a position on the occupancy, which it looks like you've got at a glance.